frontend/docs/pages/home/basics/workers.mdx

```diff
@@ -85,7 +85,7 @@ In the above examples:
 2. We register the workflow(s) that the worker is capable of executing using the `registerWorkflow` method.
 3. Finally, we start the worker process using the `start` method, allowing it to begin listening for tasks from the Hatchet engine.
 
-Run your worker process from command line with relevant environment variables set. Refer to the [quick start](https://docs.hatchet.run/home/quickstart/first-workflow) for more details on how to set up your worker.
+Run your worker process from command line with relevant environment variables set. Refer to the [quick start](https://docs.hatchet.run/self-hosting) for more details on how to set up your worker.
```

Comment:
```suggestion
Run your worker process from command line with relevant environment variables set. Refer to the following SDK guides for more details on how to set up your worker:

- [Python SDK](https://docs.hatchet.run/sdks/python-sdk/worker)
- [Typescript SDK](https://docs.hatchet.run/sdks/typescript-sdk/worker)
- [Go SDK](https://docs.hatchet.run/sdks/go-sdk/creating-a-worker)
```

I don't think `/self-hosting` is the best link here, but these should work well!